node/service/src/lib.rs

```diff
@@ -754,6 +768,19 @@ pub fn new_full<RuntimeApi, Executor>(
 		task_manager.spawn_essential_handle().spawn_blocking("babe", babe);
 	}
 
+	// Start BEEFY
+	if role.is_authority() {
```

Comment:
When the BEEFY gadget gets support for running as a full node this should get executed unconditionally.

Reply:
#40 is merged, BTW